packages/graphql/src/schema/scalars/LocalTime.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GraphQLError, GraphQLScalarType, Kind } from "graphql";
+import neo4j from "neo4j-driver";
+
+export const LOCAL_TIME_REGEX = /^(?<hour>[01]\d|2[0-3]):(?<minute>[0-5]\d):(?<second>[0-5]\d)(\.(?<fraction>\d{1}(?:\d{0,8})))?$/;
+
+export const validateLocalTime = (value: any) => {
+    if (typeof value !== "string") {
+        throw new TypeError(`Value must be of type string: ${value}`);
+    }
+
+    if (!LOCAL_TIME_REGEX.test(value)) {
+        throw new TypeError(`Value must be formatted as LocalTime: ${value}`);
+    }
+
+    return value;
+};
+
+export const parseLocalTime = (value: any) => {
+    const validatedValue = validateLocalTime(value);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const { hour, minute, second, fraction } = LOCAL_TIME_REGEX.exec(validatedValue)!.groups!;
```

Comment:
Same suggestion as on #449 - happy to discuss!

```suggestion
export const parseLocalTime = (value: any) => {
    if (typeof value !== "string") {
        throw new TypeError(`Value must be of type string: ${value}`);
    }
    
    const match = LOCAL_TIME_REGEX.exec(value);
    
    if (!match) {
        throw new TypeError(`Value must be formatted as LocalTime: ${value}`);
    }
    
    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
    const { hour, minute, second, fraction } = match.groups!;
```